<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QLCDNumber" name="result_field">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>0</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="locale">
    <locale language="German" country="Germany"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_field">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>0</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>5 * 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="equal_sign">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>8</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="keys">
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_2">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn_delete_last">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>Delete
Last Digit</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_3">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btn_sub">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_mult">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_8">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btn_div">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_6">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_point">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_4">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_5">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_7">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_1">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_0">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_add">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_9">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 233, 190);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QPushButton" name="btn_calculate">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(135, 211, 22);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="btn_clear">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
